joomla/joomla-cms
# [5.1] Remove testing channel

Pull Request for Issue #43732 .

### Summary of Changes
Remove the testing channel from CLI.


### Testing Instructions
- Try to set the testing channel via the CLI (see commands here: https://github.com/joomla/joomla-cms/issues/43732 )


### Actual result BEFORE applying this Pull Request
You can switch to the testing channel


### Expected result AFTER applying this Pull Request
You can't change to the testing channel
